marketplaceItems/avatarCustomization101/appResources/appData/avatarCustomization101.js

```diff
@@ -0,0 +1,937 @@
+            AvatarBookmarks.loadBookmark(STRING_BOOKMARK_NAME);
+            AvatarBookmarks.removeBookmark(STRING_BOOKMARK_NAME);
+            setIsAviEnabledFalse();
+        } else {
+            Window.alert("No bookmark was saved in the avatar app.");
+        }
+
+    }
+
+    function changeAvatarToAvi() {
+        // Set avatar to Avi.fst
+        MyAvatar.useFullAvatarURL(AVATAR_URL);
+        MyAvatar.setAttachmentsVariant([]);
+        setIsAviEnabledTrue();
+    }
+
+    function isAviYourCurrentAvatar() {
+        return MyAvatar.skeletonModelURL === AVATAR_URL;
+    }
+
+    // Contains all steps to set the app state to isAviEnabled = true
+    function setIsAviEnabledTrue() {
+        dynamicData.state.isAviEnabled = true;
+        spawnMirror();
+        updateUI(STRING_STATE);
+    }
+
+    // Contains all steps to set the app state to isAviEnabled = false
+    function setIsAviEnabledFalse() {
+        dynamicData.state.isAviEnabled = false;
+        dynamicData.state.activeTabName = STRING_INFO;
+
+        updateUI(STRING_STATE);
+    }
+
+    // #endregion AVATAR FUNCTIONS
+
+    // #region MATERIAL
+
+    var materialID,
+        STRING_GLASS_MAT = "glass",
+        STRING_CHAIN_MAT = "chainmail",
+        STRING_DISCO_MAT = "disco",
+        STRING_DEFAULT_MAT = "default",
+        STRING_RED_MAT = "red",
+        STRING_TEXTURE_MAT = "texture",
+        MATERIAL_GLASS = MATERIAL_DATA.glass,
+        MATERIAL_CHAINMAIL = MATERIAL_DATA.chainmail,
+        MATERIAL_DISCO = MATERIAL_DATA.disco,
+        MATERIAL_RED = MATERIAL_DATA.red,
+        MATERIAL_TEXTURE = MATERIAL_DATA.texture;
+
+    // Subtype event strings
+    var STRING_NAMED_MATERIAL_SELECTED = CONFIG.MATERIAL_EVENTS_SUBTYPE.STRING_NAMED_MATERIAL_SELECTED,
+        STRING_MODEL_TYPE_SELECTED = CONFIG.MATERIAL_EVENTS_SUBTYPE.STRING_MODEL_TYPE_SELECTED,
+        STRING_UPDATE_PROPERTY = CONFIG.MATERIAL_EVENTS_SUBTYPE.STRING_UPDATE_PROPERTY;
+
+    // componentTypes
+    var STRING_COLOR = CONFIG.STRING_COLOR,
+        STRING_MAP_ONLY = CONFIG.STRING_MAP_ONLY;
+
+    var PATH_TO_IMAGES = MATERIAL_DATA.directory;
+
+    // propertyName
+    // newMaterialData { value: "" map: "" }
+    // componentType: STRING_COLOR / STRING_SLIDER / STRING_MAP_ONLY
+    // isPBR: boolean
+    function updateMaterialProperty(propertyName, newMaterialData, componentType, isPBR) {
+
+        var value = newMaterialData.value;
+        var map = newMaterialData.map;
+        
+        var property = propertyName;
+        var propertyMap = property + "Map";
+
+        if (DEBUG) {
+            print("Update Material Property key: ", property, " value: ", value);
+            print("Update Material Property key: ", propertyMap, " map: ", map);
+        }
+        
+        // update UI values
+        var type = isPBR ? "pbr" : "shadeless";
+
+        // prep data for changing material entity in updateMaterial()
+        var updates = {
+            unlit: isPBR ? false: true
```

Comment:
```suggestion
            unlit: !isPBR
```

Reply:
Omg... That's embarrassing I didn't notice that.